dev/diffs/3.4.3.diff

```diff
@@ -961,7 +961,7 @@ index 75eabcb96f2..36e3318ad7e 100644
                _.asInstanceOf[FileScanRDD].filePartitions.forall(
                  _.files.forall(_.urlEncodedPath.contains("p=0"))))
 +        case WholeStageCodegenExec(ColumnarToRowExec(InputAdapter(
-+        fs @ CometScanExec(_, _, _, partitionFilters, _, _, _, _, _, _)))) =>
++        fs @ CometScanExec(_, _, _, _, partitionFilters, _, _, _, _, _, _)))) =>
```

Comment:
CometScanExec has an extra field (scanImpl).

It would have been better to update the syntax here to remove this "variable binding" style of matching to pull out `partitionFilters` since we can already access it as `fs.partitionFilters`. However, this would have required more work to run scalastyle and regenerate diffs for all Spark versions, so I was lazy and just added an extra `_`.